src/mysql.go

```diff
@@ -40,6 +41,10 @@ func generateDSN(args argumentList) string {
 		params = "?allowOldPasswords=true"
 	}
 
+	if args.Socket != "" {
```

Comment:
When `Socket` is passed, both `Hostname` and `Port` will be ignored. I think it would be a good idea to notify the user about this, so they're aware of some options they've specified are going to be ignored.

Coudl you add a few log lines saying this if `Socket` is defined but `Host` is not `localhost`?

Reply:
Logging added